src/Taboo/TabooTeam.py

```diff
@@ -9,9 +10,12 @@ class TabooTeam:
     """ Creates a team with a specified teamNumber
 
     """
-    def __init__(self, teamNumber):
+    def __init__(self, txQueue, teamNumber):
+        self.txQueue = txQueue
```

Comment:
Need to keep ref to txQueue? wouldn't conns be sufficient?

Reply:
No strong opinion. I'm okay with us removing this. Please feel free to make the change when you add more logic here?